docs/template.md

```diff
@@ -0,0 +1,85 @@
+# FEATURE-NAME [<sup><mark>experimental</mark></sup>](/docs/README.md#tag-notes)
+
+- Current status: ( prototype | experimental | stabilizing | stable )
+- Effort remaining:  ( little | some | lots )
```

Comment:
Maybe the right way to gauge "effort" is "how many of our six-week increments might it take"?

Reply:
If we go with this, are we building assumption that an increment is about six weeks? This could easily change though. Perhaps we can just gauge in "person-month"?